mbed_host_tests/host_tests_runner/host_test_default.py

```diff
@@ -185,6 +189,12 @@ def callback__notify_prn(key, value, timestamp):
                             self.logger.prn_inf("%s(%s)"% (key, str(value)))
                             result = value
                             break
+                        elif key == '__reset_dut':
```

Comment:
Currently only SW reset is implemented. In future value will determine SW/HW reset type.